mkdocs.yml

```diff
@@ -201,6 +217,11 @@ nav:
           - nautobot.apps.ui: "code-reference/nautobot/apps/ui.md"
           - nautobot.apps.urls: "code-reference/nautobot/apps/urls.md"
           - nautobot.apps.views: "code-reference/nautobot/apps/views.md"
```

Comment:
Does the addition of classy make the above `mkdocstrings` output irrelevant/redundant? It seems confusing to me to have two different sets of rendered docs for the same APIs.

Reply:
I don't think so... The mkdocstrings is to get that code in those files, while this provides attributes and methods within MRO in mind.